snooty/postprocess.py

```diff
@@ -1683,11 +1683,27 @@ def find_toctree_nodes(
                             else None
                         )
 
+                    isSyncPillOn = False
+
+                    # Check if tocicon is a page level option
+                    if context.pages[FileId(slug_fileid)].ast.options:
+                        if "tocicon" in context.pages[FileId(slug_fileid)].ast.options:
+                            if (
```

Comment:
I'd say we should just pass in the string, rather than checking it against "sync"